docs/business-apps/power-apps/get-started/create-your-first-custom-form.md

```diff
@@ -37,105 +40,110 @@ Before proceeding with these instructions, make sure your SharePoint site is set
 
     ![Add an image in Power Apps studio](../../../images/gs01-power-apps-studio-add-image.png)
 
-    An [Image](https://docs.microsoft.com/en-us/powerapps/maker/canvas-apps/controls/control-image) control named **Image1** will be added to the canvas and to the [Tree View](https://docs.microsoft.com/en-us/powerapps/maker/model-driven-apps/using-tree-view-on-form) on the left-hand panel.
+    An [Image](https://docs.microsoft.com/en-us/powerapps/maker/canvas-apps/controls/control-image) control named **Image1** is added to the canvas and to the [Tree View](https://docs.microsoft.com/en-us/powerapps/maker/model-driven-apps/using-tree-view-on-form) on the left-hand panel.
 
-2. To customize the look and feel of the image, you can set its properties.
-    > When a control on the canvas is selected, the Properties pane that is associated with the control opens on the right-hand panel.
+2. To customize the appearance of the image, you can set its properties.
+
+    1. When you select a control on the canvas, on the right-hand panel, the **Properties** pane associated with the control appears.
 
     ![Image control with Properties panel](../../../images/gs01-image-control-selected.png)
 
-    Since the image will be the profile photo of the owner, let us make it the same height as the **Owner_DataCard1** [data card](https://docs.microsoft.com/en-us/powerapps/maker/canvas-apps/working-with-cards).
+    2. Because the image is the profile photo of you (the owner), we recommend you make it the same height as the **Owner_DataCard1** [data card](https://docs.microsoft.com/en-us/powerapps/maker/canvas-apps/working-with-cards).
```

Comment:
The image if the profile photo of the selected item, so it probably is not the reader of this article